python/ml4ir/applications/ranking/tests/test_fixed_additive_positional_bias.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import warnings
+import tensorflow as tf
+from ml4ir.base.model.architectures.fixed_additive_positional_bias import FixedAdditivePositionalBias
+import numpy as np
+
+warnings.filterwarnings("ignore")
+
+
+class TestFixedAdditivePositionalBias(unittest.TestCase):
+
+    def one_hot_conversion(self, rank_index, max_ranks, training):
```

Comment:
@mohazahran You should name this method better. It is not just doing one hot conversion. I see asserts here as well.